metro.config.js

```diff
@@ -17,17 +18,21 @@ gutenbergMetroConfigCopy.resolver.extraNodeModules = new Proxy(
 				return gutenbergFolder;
 			}
 
-			// let's try find the module in the Jetpack submodule. We'll try the .pnpm folder.
+			// If not exists, let's try find the module in the Jetpack submodule. We'll try the .pnpm folder.
```

Comment:
Nit: want to phrase this as "If not there," or "If it doesn't exist"? Alternatively, how about replacing the whole sentence with just "Try to find the module in Jetpack's .pnpm folder instead."?

Reply:
I finally replaced it with `Try to find the module in Jetpack's .pnpm folder.` in [this commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3659/commits/2d317c0d62aa560aefb028cc79b12c965f6be909), wdyt?

Reply:
👍 works for me!